e2e/README.md

```diff
@@ -49,7 +49,7 @@ following environment variables can be exported to customize kubernetes deployme
 | ENV                  | Description                                      | Default                                                            |
 |----------------------|--------------------------------------------------|--------------------------------------------------------------------|
 | MINIKUBE_VERSION     | minikube version to install                      | latest                                                             |
-| KUBE_VERSION         | kubernetes version to install                    | v1.14.10                                                           |
+| KUBE_VERSION         | kubernetes version to install                    | latest                                                             |
```

Comment:
@agarwal-mudit is latest Kubernetes version installation is working?

Reply:
Yeah, it is working. May be there was some problem from my side when I tried it but now it is working.